assets/RNAseq_report.Rmd

```diff
@@ -424,12 +423,19 @@ if (params$input_type == "featurecounts") {
         #-->the sample name is in the file name
         files <- file.path(gsub("/$", "", params$path_gene_counts), paste0(qbicCodes, ".genes.results"))
         if (!(all(file.exists(files)))) {
-            stop("DESeq2.R could not find all of the specified .genes.results files!")
+            missing_files <- "DESeq2.R could not find all of the specified .genes.results files! Please check that for all QBiC codes in the metadata sheet, a quant.sf file is present in the gene_counts folder and make sure that no typos exist. The following file(s) could not be found: "
```

Comment:
This message says "quant.sf" files, but it is checking for "<QBiC-code>.genes.results" files. Should it be updated to this: `<QBiC-code>.genes.results`.